src/tests/system/tests/test_sssctl.py

```diff
@@ -545,3 +545,292 @@ def test_sssctl__analyze_with_logdir_and_non_root_privileges(client: Client, pro
     assert result_root.rc == 0, "analyze failed as root with --logdir"
     assert result_user.rc == 0, "analyze failed as user with --logdir"
     assert result_root.stdout == result_user.stdout, "Root and user output mismatch with --logdir"
+
+
+@pytest.mark.parametrize(
+    "use_fqdn,case_sensitive",
+    [
+        (False, True),
+        (True, True),
+        (True, False),
+    ],
+    ids=["shortnames", "fqdn", "fqdn_case_insensitive"],
+)
+@pytest.mark.tools
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__user_show(client: Client, provider: GenericProvider, use_fqdn: bool, case_sensitive: bool):
```

Comment:
Let's break this test into two

test_sssctl_user_show

test_sssctl_user_fully_qualified_name_show **or**
test_sssctl_user_show_fully_qualified_name 

This matches other suites.

Reply:
done